kolena/initialize.py

```diff
@@ -36,23 +41,51 @@ def initialize(
     """
     Initialize a client session.
 
+    A session has a global scope and remains active until interpreter shutdown.
+
     Retrieve an API token from the [:kolena-developer-16: Developer](https://app.kolena.io/redirect/developer) page and
-    populate the `KOLENA_TOKEN` environment variable before initializing:
+    make it available through one of the following options before initializing:
 
+
+    1. Directly through the optional `api_token` argument
     ```python
     import os
     import kolena
 
     kolena.initialize(os.environ["KOLENA_TOKEN"], verbose=True)
     ```
+    2. Populate the `KOLENA_TOKEN` environment variable
+    ```bash
+    export KOLENA_TOKEN="********"
+    ```
+    3. Store in [.netrc file](https://www.gnu.org/software/inetutils/manual/html_node/The-_002enetrc-file.html)
+    ```cfg title="~/.netrc"
+    machine api.kolena.io password ********
+    ```
 
-    A session has a global scope and remains active until interpreter shutdown.
+    ```mermaid
+    flowchart TD
+        Start[Get API token]
+        Step1{{`api_token` argument provided?}}
```

Comment:
Is there a way for us to get this to render in Mermaid as monospaced text, rather than as normal text wrapped in literal backticks?

Reply:
I wanted the same thing but wasn't able to get it to work😞